README.md

```diff
@@ -1089,22 +1089,34 @@ Other Style Guides
     export default es6;
     ```
 
-  <a name="modules--no-wildcard"></a><a name="10.2"></a>
-  - [10.2](#modules--no-wildcard) Do not use wildcard imports.
-
-    > Why? This makes sure you have a single default export,
-    > and forces you to define the API (keeping things organized)
+  <a name="modules--define-api"></a><a name="10.2"></a>
+  - [10.2](#modules--define-api) Always define your module's API at the bottom of the file.
```

Comment:
Ah.  In most cases when you are exporting only 1 thing (like a class or a function), it should be a default export.

In `selectors/index.js` this is a special case in which we expect that future work will warrant the use of the exported object list and including that syntax now allows for minimal impact when future items are added.